### PR TITLE
[task_panlels] Support for Kibiter/Kibana 6.x

### PR DESCRIPTION
**Pull request by jgbarah**
Changes to interaction with Elasticsearch and Kibiter for supporting 6.x in both of them.

Main changes are in the following areas:

* Detection of the version of Kibiter/Kibana
* Uploading of Kibiter/Kibana configuration properties (such as default index pattern)
* Uploading of the main menu for the dashboard
* Use of mandatory header for Elasticsearch 6.x

This patch relies on other pathches to Kidash and GrimoireELK for supporting Elasticsearch and Kibiter/Kibana 6.x.

### CODE REVIEW

**Comment by jgbarah** [comment]
Fixed flake8 errors.

File: mordred/task_panels.py [hunk 1]
```diff
@@ -76,38 +96,101 @@ def __init__(self, conf):
...
-        config_url = '.kibana/config/' + kibiter_version
+        kibiter_version = self.__kibiter_version(kibiter_major)
+        print("Kibiter/Kibana: version found is %s" % kibiter_version)
+        time_picker = "{\n  \"from\": \"" + kibiter_time_from \
+            + "\",\n  \"to\": \"now\",\n  \"mode\": \"quick\"\n}"
+
+        if kibiter_major == "6":
+            config_resource = '.kibana/doc/config:' + kibiter_version
+        else:
+            config_resource = '.kibana/doc/config:' + kibiter_major
```
Comment (acs) [inline mordred/task_panels.py]: This is not correct for Elasticsearch 2.x and 5.x

**Comment by acs** [comment]
Ok, I will allow this PR and the I will do a "hot fix" to solve the issue with ES 2.x and 5.x versions. The patch is pretty easy:

```
diff --git a/mordred/task_panels.py b/mordred/task_panels.py
index 72cc28b..4c52215 100644
--- a/mordred/task_panels.py
+++ b/mordred/task_panels.py
@@ -177,7 +177,7 @@ class TaskPanels(Task):
         if kibiter_major == "6":
             config_resource = '.kibana/doc/config:' + kibiter_version
         else:
-            config_resource = '.kibana/doc/config:' + kibiter_major
+            config_resource = '.kibana/config/' + kibiter_version
         kibiter_config = {
             "defaultIndex": kibiter_default_index,
             "timepicker:timeDefaults": time_picker
```

**Comment by jgbarah** [comment]
Thanks for the fix!!!

**Pull request merged**